x-pack/plugins/security/server/routes/api/v1/authenticate.js

```diff
@@ -31,9 +45,8 @@ export function initAuthenticateApi(server) {
       const { username, password } = request.payload;
 
       try {
-        const authenticationResult = await server.plugins.security.authenticate(
-          BasicCredentials.decorateRequest(request, username, password)
-        );
+        request.loginAttempt.setCredentials(username, password);
+        const authenticationResult = await server.plugins.security.authenticate(request);
```

Comment:
Rather than extracting the username and password from the payload and faking a basic auth header on the request to force a new login during `authenticate`, login attempts in `authenticate` will now be handled exclusively through the new `loginAttempt` object on the request.